lightfm/evaluation.py

```
@@ -232,14 +235,14 @@ def auc_score(
 
     assert np.all(ranks.data >= 0)
 
-    auc = np.zeros(ranks.shape[0], dtype=np.float32)
+    auc = create_zeros(ranks.shape[0])
 
     if train_interactions is not None:
         num_train_positives = np.squeeze(
```

Pylint (3.3.7)

[error] Undefined variable 'np'

(E0602)